src/modules/hip/kernel/resize_mirror_normalize.hpp

```diff
@@ -0,0 +1,369 @@
+                               roiTensorPtrSrc);
+        }
+        else if ((srcDescPtr->layout == RpptLayout::NCHW) && (dstDescPtr->layout == RpptLayout::NCHW))
+        {
+            hipLaunchKernelGGL(resize_mirror_normalize_bilinear_pln_tensor,
+                               dim3(ceil((float)globalThreads_x/localThreads_x), ceil((float)globalThreads_y/localThreads_y), ceil((float)globalThreads_z/localThreads_z)),
+                               dim3(localThreads_x, localThreads_y, localThreads_z),
+                               0,
+                               handle.GetStream(),
+                               srcPtr,
+                               make_uint3(srcDescPtr->strides.nStride, srcDescPtr->strides.cStride, srcDescPtr->strides.hStride),
+                               dstPtr,
+                               make_uint3(dstDescPtr->strides.nStride, dstDescPtr->strides.cStride, dstDescPtr->strides.hStride),
+                               dstImgSizes,
+                               dstDescPtr->c,
+                               handle.GetInitHandle()->mem.mgpu.floatArr[0].floatmem,
+                               handle.GetInitHandle()->mem.mgpu.floatArr[1].floatmem,
+                               handle.GetInitHandle()->mem.mgpu.uintArr[2].uintmem,
+                               roiTensorPtrSrc);
+        }
+        else if ((srcDescPtr->c == 3) && (dstDescPtr->c == 3))
+        {
+            if ((srcDescPtr->layout == RpptLayout::NHWC) && (dstDescPtr->layout == RpptLayout::NCHW))
+            {
+                hipLaunchKernelGGL(resize_mirror_normalize_bilinear_pkd3_pln3_tensor,
+                                   dim3(ceil((float)globalThreads_x/localThreads_x), ceil((float)globalThreads_y/localThreads_y), ceil((float)globalThreads_z/localThreads_z)),
+                                   dim3(localThreads_x, localThreads_y, localThreads_z),
+                                   0,
+                                   handle.GetStream(),
+                                   srcPtr,
+                                   make_uint2(srcDescPtr->strides.nStride, srcDescPtr->strides.hStride),
+                                   dstPtr,
+                                   make_uint3(dstDescPtr->strides.nStride, dstDescPtr->strides.cStride, dstDescPtr->strides.hStride),
+                                   dstImgSizes,
+                                   handle.GetInitHandle()->mem.mgpu.floatArr[0].floatmem,
+                                   handle.GetInitHandle()->mem.mgpu.floatArr[1].floatmem,
+                                   handle.GetInitHandle()->mem.mgpu.uintArr[2].uintmem,
+                                   roiTensorPtrSrc);
+            }
+            else if ((srcDescPtr->layout == RpptLayout::NCHW) && (dstDescPtr->layout == RpptLayout::NHWC))
+            {
+                hipLaunchKernelGGL(resize_mirror_normalize_bilinear_pln3_pkd3_tensor,
```

Comment:
Isn't there any issue with this kernel? Not sure if the kernel launch would be correct here.

Reply:
Number of threads was more than what was required in this case but since we added the below condition check there is nod discrepancy observed with outputs
if ((id_y >= dstDimsWH.y) || (id_x >= dstDimsWH.x))
    {
        return;
    }
    
Made the change as globalThreads_x = (dstDescPtr->w + 7) >> 3;